apps/web/src/serlo-editor/plugins/text/types/text-editor.ts

```diff
@@ -12,13 +12,23 @@ export type CustomElement =
   | MathElement
   | Blank
 
+export interface Answer {
+  answer: string
+  // Here we could store feedback for the learner that shows up when answers are checked
+  //learnerFeedback?: string
+}
+
 export interface Blank {
   type: 'blank'
   children: CustomText[]
   blankId: string // Used to uniquely identify a blank
-  correctAnswer: string
-  // Disabled alternative correct solutions for now
-  // alternativeSolutions: string[]
+  correctAnswers: Answer[]
+  // Here we could specify incorrect answers for this blank with specific learner feedback
+  // incorrectAnswers?: Answer[]
+  // Here we could add a default feedback for the learner
+  // defaultLearnerFeedback: string
+  // Here we could specify a method to check the answer
+  // checkAnswerMethod?: 'text' | 'number' | 'math-expression'
```

Comment:
```suggestion
  // checkAnswerMethod?: 'text' | 'number' | 'math-expression'
```

* Maybe this should be merged with `type` to have `blank | blank-number | blank-math-expression`. -> Depends how the blank type should be changed. The current model makes sense when you first add a blank and then you configure the blank to a number blank. A change would be better when you can directely add a number blank in the editor...
* Type of `Answer` will be different depending of the type of the blank. For example:

```typescript
interface BlankBase {
  children: CustomText[]
  blankId: string
}

interface TextBlank extends BlankBase {
  type: "text-blank"
  correctAnswers: TextAnswer[]
}

interface NumberBlank extends BlankBase {
  type: "number-blank"
  answer: number
}
```

Same can be done with `checkAnswerMethod` btw... 

```typescript
interface BlankBase {
  type: "blank"
  children: CustomText[]
  blankId: string
}

interface TextBlank extends BlankBase {
  blankType: "text"
  correctAnswers: TextAnswer[]
}

interface NumberBlank extends BlankBase {
  blankType: "number"
  answer: number
}
```

Reply:
I like this idea very much! :heart:  
I propose we remove `checkAnswerMethod` and define the type of the blank and the method for checking the answer through `type: 'text-blank' | 'number-blank' | 'math-blank'`
The editor toolbar could show one button to create a new blank (per default 'text-blank') and another UI element to switch the type when the blank is selected.